configs/DAKEFPVH743/config.h

```diff
@@ -121,9 +121,9 @@
     TIMER_PIN_MAP( 13, PB9,  1, -1) \
     TIMER_PIN_MAP( 14, PE9 , 1,  8)
 
-#define ADC2_DMA_OPT        9
+#define ADC1_DMA_OPT        8
+#define ADC3_DMA_OPT        9
```

Comment:
This would conflict with PE9
```suggestion
#define ADC1_DMA_OPT        9
#define ADC3_DMA_OPT        10
```